src/models/classes/order.js

```diff
@@ -5,11 +5,13 @@ class Order
      *
      * @param products
      */
-    constructor(products)
+    constructor(products, time = null, id = null)
```

Comment:
I agree. However, we also use this model for the order of the customer.
So we can set `time` as default value, but it might get confusing for `id`.